professional/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from user.models import Profile
+# from reservation.model import Appointment, TypeOfJob, Schedule
+
+
+class Professions(models.TextChoices):
+    Plumber = 'PLU', 'Plumber'
+    Gardener = 'GAR', 'Gardener'
+    Electrician = 'ELE', 'Electrician'
+    Tinsmith = 'TIN', 'Tinsmith'
+    Painter = 'PAI', 'Painter'
+    Locksmith = 'LOC', 'Locksmith'
+    Exterminator = 'EXT', 'Exterminator'
+    GasTechnician = 'GAT', 'GasTechnician'
+    AirConditioningTechnician = 'ACT', 'AirConditioningTechnician'
+    RefrigeratorTechnician = 'RET', 'RefrigeratorTechnician'
+    Cleaner = 'CLE', 'Cleaner'
+    Handyman = 'HAN', 'Handyman'
+
+
+class Professional(models.Model):
+    professional_id = models.BigAutoField(primary_key=True)
+    profile_id = models.ForeignKey(Profile, on_delete=models.CASCADE)
+    profession = models.CharField(max_length=3, choices=Professions.choices, blank=True)
+    description = models.TextField()
+    # schedule = models.ForeignKey(Schedule, on_delete=models.CASCADE, default=None)
```

Comment:
You do not need to add this field.
It is not exist in the ERD

Reply:
Done